src/VTUFiles.f90

```diff
@@ -472,17 +473,19 @@ SUBROUTINE close_VTUXMLFileType(myVTKFile)
 !> Writes the pvtu file for visualizing in parallel (only if master rank and
 !> the number of processors is greater than one).
 !>
-    SUBROUTINE writepvtu_VTUXMLFileType(fileobj,funit,filen,procs,rank)
+    SUBROUTINE writepvtu_VTUXMLFileType(fileobj,funit,case,filen,procs,rank)
       CHARACTER(LEN=*),PARAMETER :: myName='writepvtu_VTUXMLFileType'
       CLASS(VTUXMLFileType),INTENT(INOUT) :: fileobj
-      CHARACTER(LEN=*),INTENT(IN) :: filen
+      CHARACTER(LEN=*),INTENT(IN) :: case,filen
       CHARACTER(LEN=128) :: sint
       INTEGER(SIK),INTENT(IN) :: funit,procs,rank
       INTEGER(SIK) :: i,j,iord
       CHARACTER(LEN=128) :: fname,fmtStr
       !
       IF((procs>1).AND.rank == 0) THEN
-        OPEN(unit=funit,file=TRIM(filen)//'.pvtu')
+        sint='fsr_'//TRIM(case)//'/'
```

Comment:
Maybe a little picky... but if futility is supposed to be a generic library, should we hard-code fsr into the name?  Its pretty MPACT specific.